docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -309,14 +309,18 @@ Unit    Description
 
 .. function:: human_readable_seconds(double) -> varchar
 
-    Returns ``seconds`` expressed in terms of ``human readable interval``::
+    Parses value of ``seconds`` into a human readable varchar containing
```

Comment:
Maybe 

Parses the double value of seconds into a human readable string containing ...

Reply:
Agreed

Reply:
Instead of "parses", I would use "formats".

Reply:
Suggestions have been implemented @mosabua. I emailed a signed CLA to cla@trino.io on Tuesday at 12:47 AM EDT. Is there anything else I need to do for the CLA?

![Trino Screenshot 2](https://user-images.githubusercontent.com/22308076/117241370-05c61500-ae01-11eb-9355-643f5a26ae13.png)